Quick/Configuration/QuickConfiguration.m

```diff
@@ -1,7 +1,10 @@
 #import "QuickConfiguration.h"
+#import "World.h"
 #import <Quick/Quick-Swift.h>
 #import <objc/runtime.h>
 
+QuickWorldBridge
```

Comment:
You just need to add `#import <Quick/Quick-Swift.h>` inside `World.h` to get the `SWIFT_CLASS` macro.